bentoml/_internal/configuration/containers.py

```diff
@@ -62,17 +68,15 @@
         "logging": {
             "level": And(
                 str,
-                lambda level: level.isupper(),
+                _is_upper,
                 error="logging.level must be all upper case letters",
             ),
             "console": {"enabled": bool},
             "file": {"enabled": bool, "directory": Or(str, None)},
             "advanced": {"enabled": bool, "config": Or(dict, None)},
         },
         "tracing": {
-            "type": Or(
-                And(str, Use(str.lower), lambda s: s in ("zipkin", "jaeger")), None
-            ),
+            "type": Or(And(str, Use(str.lower), _check_tracing_type), None),
```

Comment:
this case is better left it is, easier to read in context

Reply:
This mainly to compensate pyright :(

Reply:
can't believe there's no easy way to type a lambda function inline :(